visualization.py

```diff
@@ -1,10 +1,18 @@
 import matplotlib.pyplot as plt
+import data_management
```

Comment:
Do you use any functions or variables from `data_management` ?
If not, you can remove this line.